### PR TITLE
Update to moving-flow-SE

### PR DESCRIPTION
**Pull request by zakkjellin**
First iteration of the Swedish moving flow! Yay!!!

### CODE REVIEW

File: angel-data/moving-flow-SE.json [hunk 1]
```diff
@@ -0,0 +1,499 @@
+{
+  "id": "0e9214ae-d578-43b1-8e67-cdbb5aeddfd3",
+  "ifid": "29F1BC05-1702-4DA0-B866-9139E5B2C621",
+  "keywords": {
+    "isTrusted": true
+  },
```
Comment (gustaveen) [inline angel-data/moving-flow-SE.json:6]: @hedvigoscar @sampettersson is this a new keyword you added? or do you know where it comes from? :) keywords should only be strings right? and this one seems to pop up when ever I create a new story in angel 😅

Comment (hedvigoscar) [reply angel-data/moving-flow-SE.json:6]: No idea unfortunately 😞

Comment (gustaveen) [reply angel-data/moving-flow-SE.json:6]: The mystery continues! 🕵️‍♂️

Comment (styrelius) [reply angel-data/moving-flow-SE.json:6]: @gustaveen Won't this `isTrusted` make this flow break? So this flow needs to be remade or? 

Also - I think when it comes to the moving flow stuff it makes most sense that @sampettersson and @hedvigoscar does the approving right?

Comment (sampettersson) [reply angel-data/moving-flow-SE.json:6]: @gustaveen weird... went into Angel imported the file and opened the keywords modal and clicked save and it removed it?!

Screenshot 2021-03-25 at 19 23 15

@styrelius yeah, I agree, makes most sense if we review these 😄

File: angel-data/moving-flow-SE.json [hunk 2]
```diff
@@ -0,0 +1,499 @@
+{
+  "id": "0e9214ae-d578-43b1-8e67-cdbb5aeddfd3",
+  "ifid": "29F1BC05-1702-4DA0-B866-9139E5B2C621",
+  "keywords": {
+    "isTrusted": true
```
Comment (sampettersson) [inline angel-data/moving-flow-SE.json:5]:
```suggestion
```

File: angel-data/moving-flow-SE.json [hunk 3]
```diff
@@ -0,0 +1,498 @@
+{
+  "id": "0e9214ae-d578-43b1-8e67-cdbb5aeddfd3",
+  "ifid": "29F1BC05-1702-4DA0-B866-9139E5B2C621",
+  "keywords": {
```
Comment (sampettersson) [inline angel-data/moving-flow-SE.json:4]:
```suggestion
  "keywords": {},
```

File: angel-data/moving-flow-SE.json [hunk 4]
```diff
@@ -0,0 +1,498 @@
...
+  },
```
Comment (sampettersson) [inline angel-data/moving-flow-SE.json:5]:
```suggestion
```

**Review by sampettersson — APPROVED** [review]

**Pull request merged**